packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -155,7 +155,7 @@ const NonEmptyCell = ({
             undefined,
             scopedSchema.type === 'object' ? `#/properties/${propName}` : '#'
           )}
-          path={rowPath}
+          path={rowPath + (scopedSchema.type === 'object' ? '.' + propName : '')}
```

Comment:
good catch, we should probably fix line 146 accordingly

Reply:
Fixed.